<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>614</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>611</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>530</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="mainpage_tab">
     <property name="minimumSize">
      <size>
       <width>994</width>
       <height>501</height>
      </size>
     </property>
     <attribute name="title">
      <string>MAIN PAGE</string>
     </attribute>
     <widget class="QLineEdit" name="search_lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_button">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>110</y>
        <width>256</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Reader category</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>For children</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>For students</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>For adults</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Genre</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fantasy</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thriller</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Novel</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fiction</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Publishing house</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AST</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Exmo</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Book</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minimal price</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="applyFilters_button">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>550</y>
        <width>251</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>APPLY FILTERS</string>
      </property>
     </widget>
     <widget class="QListWidget" name="stock_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>731</width>
        <height>471</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sort_combobox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>216</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>СОРТИРОВАТЬ ПО</string>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <item>
       <property name="text">
        <string>СОРТИРОВАТЬ ПО</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>СНАЧАЛА НОВИНКИ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО АВТОРУ (А -&gt; Я)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО АВТОРУ (Я -&gt; А)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО НАЗВАНИЮ (А -&gt; Я)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО НАЗВАНИЮ (Я -&gt; А)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО РЕЙТИНГУ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО ЦЕНЕ (СНАЧАЛА ДЕШЕВЫЕ)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПО ЦЕНЕ (СНАЧАЛА ДОРОГИЕ)</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="purchasehistory_tab">
     <property name="minimumSize">
      <size>
       <width>994</width>
       <height>501</height>
      </size>
     </property>
     <attribute name="title">
      <string>PURCHASE HISTORY</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="cart_tab">
     <property name="minimumSize">
      <size>
       <width>994</width>
       <height>501</height>
      </size>
     </property>
     <attribute name="title">
      <string>CART</string>
     </attribute>
     <widget class="QTableWidget" name="cart_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>631</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="totalPrice_label">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_lineEdit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneNumber_lineEdit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>phone number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>130</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="cartError_label">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="order_button">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>190</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ORDER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>330</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rub</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wishlist_tab">
     <property name="minimumSize">
      <size>
       <width>994</width>
       <height>501</height>
      </size>
     </property>
     <attribute name="title">
      <string>WISH LIST</string>
     </attribute>
     <widget class="QTableWidget" name="wishList_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>631</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="reportAdmission_button">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>190</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>REPORT ADMISSION</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneNumber_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>phone number</string>
      </property>
     </widget>
     <widget class="QLabel" name="wishlistError_label">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>130</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PublishersDeliveryOrders_tab">
     <attribute name="title">
      <string>Publishers delivery orders</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
